edge-middleware/feature-flag-split/app/about/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { cookies } from 'next/headers'
+import { AboutA } from './a'
+import { AboutB } from './b'
+import { createSplitClient } from '@lib/split'
+
+// export const runtime = 'edge'
```

Comment:
nit: leftover comment

Reply:
There is an unknown issue in this repo where dev mode breaks when I comment this out. But somehow it works when using `export const config = { runtime: 'edge' }` instead. This is kind of a reminder for that since `export const config` should no longer be used.